src/rules/font-family-name-quotes/__tests__/index.js

```diff
@@ -96,6 +98,21 @@ testRule(rule, {
   }, {
     code: "a { font-family: -apple-system, 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', sans-serif; }",
     message: messages.rejected("BlinkMacSystemFont"),
+  }, {
+    code: "a { font: italic 300 16px/30px Arial, serif; }",
+    message: messages.expected("Arial"),
+    line: 1,
+    column: 32,
+  }, {
```

Comment:
> I think unknown numerical font weights get ignored, like other invalid CSS values.

👍 

> I do think that rule would fit with the other no-unknown rules.

Will create a new issue for a new rule: `font-weight-notation-no-unknown`